docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
```

Comment:
@jimmidyson Once you're done with it can I make it pretty like the second diagram I made for James S? I'll color-coordinate it for you and everything. :D It's probably going to go on the iPaaS website once I'm done "revamping" it this week. lol